pipeline.py

```diff
@@ -51,19 +63,28 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+
+        files = list(self.path_to_raw_txt_data.glob('*_raw.txt'))
```

Comment:
no reason to create a list from glob iterable, right?

Reply:
I was probably thinking of doing something list-specific, but now there's no point.